frontend/onied/src/components/supportChatForUser/supportChatWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import ChatButton from "@onied/components/supportChatForUser/chatButton/chatButton";
+import ChatWindow from "@onied/components/supportChatForUser/chatWindow/chatWindow";
+import { MessagesHistoryDto } from "@onied/components/supportChatForUser/messageDtos";
+import GetHistory from "@onied/components/supportChatForUser/tempMessagesSource";
+
+function SupportChatWrapper() {
+  const [isChatWindowOpen, setIsChatWindowOpen] = useState(false);
+
+  const [messagesHistory, setMessagesHistory] = useState<MessagesHistoryDto>();
+
+  useEffect(() => {
+    setMessagesHistory(GetHistory(0));
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    api
      .get("/chat")
      .then((response) => {
        setMessagesHistory(response.data);
      })
      .catch((error: AxiosError<any, any>) => {
        if (error.response != null) {
          if (error.response.status == 404) {
            // Чат для пользователя еще не был создан; создастся при первом сообщении.
            // Отобразить вместо чата что-то по-типу текста "Введите вопрос, чтобы начать"
          } else {
            // Не показывать чат вообще
          }
        } else {
          // ВРЕМЕННОЕ РЕШЕНИЕ, ТОЛЬКО ДЛЯ ТЕСТОВ
          console.error("Could not reach the server.");
          setMessagesHistory(GetHistory(0));
          // В нормальной ситуации просто не показывать чат
        }
      });
  }, []);
```

Примерно такой код здесь должен быть, естественно вместо комментариев нормальный код. Прошу доделать